src/main/java/seedu/duke/UserInterface.java

```diff
@@ -34,70 +35,25 @@ public void run() {
                 switch (command) {
                 case "view":
                     //using flags here to distinguish between different views????
-                    String regex = "(?<flag>[og])/ id/(?<id>\\d*)";
-                    Regex regexMatch = new Regex(userInput, regex);
-                    HashMap<String, String> matches = regexMatch.getGroupValues();
-                    if (matches.get("flag").equals("o")) {
-                        // view order with flag "o/"
-                        warehouse.viewOrder(matches.get("id"));
-                    } else if (matches.get("flag").equals("g")) {
-                        // view good with flag "g/"
-                        warehouse.viewGood(matches.get("id"));
-                    } else {
-                        // wrong command exception
-                        throw new WrongCommandException("view", true);
-                    }
+                    userInputIsView(userInput);
```

Comment:
Suggest change to "parseViewCmd(userInput)". Also due to commonality in certain code, would prefer if these are classes that inherit from a base class like "CommandsParseer.java"